partitionmanager/cli.py

```diff
@@ -253,6 +256,13 @@ def bootstrap_cmd(args):
     nargs="+",
     help="table names, overwriting config",
 )
+BOOTSTRAP_PARSER.add_argument(
+    "--assume-partitioned-on",
+    type=partitionmanager.types.SqlInput,
+    action="append",
+    help="Assume tables are partitioned by this column name, can be specified "
+    "multiple times for multi-column partitions",
+)
```

Comment:
Maybe, but I also don't expect this to be called more than a single time per table, ever, as it is only to help avoid typos during the otherwise repetitive construction of these triggers.

Nevertheless, let's tackle that in both #17 and #18 .